tutorials/nemo-retriever-synthetic-data-generation/README.md

```diff
@@ -0,0 +1,108 @@
+# NeMo Retriever Synthetic Data Generation
+
+NeMo Retriever Synthetic Data Generation (SDG) is designed to streamline the creation of high-quality evaluation datasets for Text QA retrieval use cases. By leveraging existing enterprise data, this pipeline enables rapid generation of relevant evaluation datasets, facilitating improved model performance.
+
+This version supports the generation of evaluation datasets, creating synthetic benchmark datasets compatible with commonly used evaluation frameworks such as [BEIR](https://huggingface.co/datasets/BeIR/beir). Synthetic training dataset generation will be supported in an upcoming version.
+
+NeMo Retriever SDG can be run either from the command line, or using the [notebook example](notebooks/quickstart.ipynb) provided in this repository. Check the [Prerequisites](#prerequisites) section for instructions on generating an API key and installing libraries. To get started with the notebook, follow the [Notebook Quick Start](#run-pipeline-ipython-notebook) instructions. Otherwise, follow the [CLI Quick Start](#run-pipeline-cli) section.
+
+![NeMo Retriever SDG](figures/sdg_pipeline.png)
+
+#### Key Features
+
+* Quickly generate complex QA datasets from existing text documents for retriever model evaluation.
+* Output datasets can be formatted in [SQuAD (Stanford Question Answering Dataset)](https://huggingface.co/datasets/rajpurkar/squad) or [BEIR (Benchmarking Information Retrieval)](https://huggingface.co/datasets/BeIR/beir) format for easy integration into evaluation workflows.
+* Designed to integrate seamlessly with [NVIDIA NeMo Evaluator](https://developer.nvidia.com/nemo-microservices) microservice, currently in early access.
+
+
+## Quickstart
+
+### Prerequisites
+
+In order to use NeMo Retriever SDG, you will need access to NVIDIA’s API Catalog. Go to the [NGC Personal Key Manager](https://org.ngc.nvidia.com/setup) to generate a Personal Key that will allow you to access AI Foundation Models and Endpoints.
+
+To install the required libraries, navigate to the root directory of the project and run the following command in your notebook or command line:
+
+```
+$ pip install -r requirements.txt
+```
+
+Alternatively, you can use container `nvcr.io/nvidia/pytorch:24.01-py3`.
```

Comment:
Can you recommend users use the latest NeMo Framework container instead of the PyTorch one? See list of containers here: https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo. `nvcr.io/nvidia/nemo:24.09` should be good.